anitya/lib/backends/maven.py

```diff
@@ -70,3 +74,50 @@ def get_versions(cls, project):
                           .replace(':', '/'))
 
         return get_versions_by_regex(url, REGEX, project)
+
+    @classmethod
+    def check_feed(cls):
+        '''check_feed for Maven backend.
+
+        Return a generator over the latest 40 packages to Maven central index by
+        calling maven-index-checker application
+
+        Returns:
+            generator over new packages
+        '''
+        maven_url = 'http://repo2.maven.org/maven2'
+        if APP.config['JAVA_PATH'] is None:
+            raise AnityaPluginException('no java binary specified')
+        if APP.config['JAR_NAME'] is None:
+            raise AnityaPluginException('no maven-release-checker jar file specified')
+
+        try:
+            data = check_output([APP.config['JAVA_PATH'], "-jar",
+                                 APP.config['JAR_NAME']], universal_newlines=True)
+        except CalledProcessError:
+            raise AnityaPluginException('maven-release-checker exited with non zero value')
+
+        data = json.loads(data)
+        session = init(APP.config['DB_URL'])
```

Comment:
Sorry I missed this earlier - this is why the database tables aren't present in your tests. This function creates a new SQLAlchemy engine and session factory using the app's configured database, but the tests use an in-memory SQLite database it sets up for you. 

I've [got a PR open](https://github.com/release-monitoring/anitya/pull/466) to make this whole pattern much simpler. When that's merged, all you need to do is:
```
Project.query.filter(Project.version_url == maven_coordinates).all()
```
The query property takes care of creating a query object from the scoped session automatically and you don't have to worry about it.

I'll get that PR passing CI and badger someone to review it ASAP, then you can rebase, modify your query here, and hopefully the tests should start passing!